src/Components/ImportHAR.jsx

```diff
@@ -1,29 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDropzone } from 'react-dropzone';
+import { Button } from 'react-bootstrap';
+import { useNetwork } from '../state/network/provider';
+
+import Styles from './ImportHAR.module.scss';
 
 const DROP_FILE_CONFIG = {
   accept: '.har',
   multiple: false,
 };
 
-const ImportHar = ({ onDataLoad, onError }) => {
+const ImportHar = ({ showButton }) => {
+  const { actions } = useNetwork();
+
+  const prepareData = newNetworkData => (
```

Comment:
` const prepareData = (newNetworkData) => (`

just to keep consistent

Reply:
`eslint` doesn't allow me that, because of the return `(` in the same line.